llvm/lib/Target/AMDGPU/AMDGPUSubtarget.h

```diff
@@ -377,6 +378,8 @@ class AMDGPUSubtarget {
   uint64_t getExplicitKernArgSize(const Function &F, Align &MaxAlign) const;
   unsigned getKernArgSegmentSize(const Function &F, Align &MaxAlign) const;
 
+  unsigned getPrefLoopAlignment() const { return PrefLoopAlignmentLog2; }
```

Comment:
Do we really need a target feature for this? Can we just have the getter return the correct value based on gfx version directly below?

Reply:
I'm more wondering if this is a universal property that should have always been done. The prior comment says there's no benefit pre-gfx10, so why is this now needed for gfx9?

Reply:
> The prior comment says there's no benefit pre-gfx10, so why is this now needed for gfx9?

Yes, same doubt.